GPU/GLES/Framebuffer.cpp

```diff
@@ -1305,8 +1321,8 @@ void FramebufferManager::BlitFramebuffer(VirtualFramebuffer *dst, int dstX, int
 	}
 	int srcX1 = srcX * srcXFactor;
 	int srcX2 = (srcX + w) * srcXFactor;
-	int srcY2 = src->renderHeight - (h + srcY) * srcYFactor;
-	int srcY1 = srcY2 + h * srcYFactor;
+	int srcY2 = srcY * srcYFactor;
+	int srcY1 = (srcY + h) * srcYFactor;
```

Comment:
Oh! Good spot, I was staring at the code too long to see it... I'll try it soon.